com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -249,39 +247,94 @@ private void OnValidate()
                 };
             }
 
+            //If true, check to see if there is a defined player prefab in the Network prefab list
+            //This helps migrate v0.1.0 projects (i.e. no PlayerPrefab was assigned) as well as helps detect
+            //specific interface related issues related to the NetworkPrefabs list.
+            var scanForPlayerPrefab = NetworkConfig.PlayerPrefab == null;
+
+            //During OnValidate we will always clear out NetworkPrefabOverrideLinks and rebuild it in order to
+            //detect specific edge case scenarios (See NtworkConfig.PreviousPlayerPrefab comments)
+            NetworkConfig.NetworkPrefabOverrideLinks.Clear();
+
+            //Check network prefabs and assign to dictionary for quick look up
             for (int i = 0; i < NetworkConfig.NetworkPrefabs.Count; i++)
             {
                 if (NetworkConfig.NetworkPrefabs[i] != null && NetworkConfig.NetworkPrefabs[i].Prefab != null)
                 {
-                    if (NetworkConfig.NetworkPrefabs[i].Prefab.GetComponent<NetworkObject>() == null)
+                    var networkObject = NetworkConfig.NetworkPrefabs[i].Prefab.GetComponent<NetworkObject>();
+                    if (networkObject == null)
                     {
                         if (NetworkLog.CurrentLogLevel <= LogLevel.Normal)
                         {
                             NetworkLog.LogWarning($"{nameof(NetworkPrefab)} [{i}] does not have a {nameof(NetworkObject)} component");
                         }
                     }
-                }
-            }
+                    else
+                    {
+                        //If someone is transitioning to this new format, then go ahead and populate the PlayerPrefab reference for them.
+                        if (scanForPlayerPrefab && NetworkConfig.NetworkPrefabs[i].IsPlayer)
+                        {
+                            //If we find a NetworkPrefab where IsPlayer is true and we have
+                            //no PlayerPrefab nor did we have one previously (i.e. we didn't just delete it),
+                            //then we want to set the default player prefab.  We also want to set the
+                            //previous player prefab value in the event the user decides to delete the
+                            //PlayerPrefab value from within the editor inspector view.
+                            //(See NtworkConfig.PreviousPlayerPrefab comments for more information)
```

Comment:
nit: typo.  Also I think the preferred comment style has a space between the // and the text

Reply:
resolved in [7f2781c](https://github.com/Unity-Technologies/com.unity.multiplayer.mlapi/pull/749/commits/7f2781cb3913a689abfbdb1c6bc977b42fbf589b)

Made sure all added comments (and areas surrounding within reason) have spaces between the forward slashes and the text body of the comment.